src/components/common/CapitalLink.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <component :is="element" class="capital-link">{{ text }}</component>
+</template>
+<script>
+export default {
+  props: {
+    text: { type: String, required: true },
+    href: { type: String, default: "" },
+  },
+  setup(props) {
+    const element = props.href ? "a" : "span";
+    return { element };
```

Comment:
You might want to make it a computed, just to be safe - this way if the prop would change, your component would deal with it without hickups! Otherwise it would only have the initial value 💡 

```suggestion
    const element = computed(() => props.href ? "a" : "span");
    return { element };
```